packages/plugins/whisper/src/index.js

```diff
@@ -38,6 +50,57 @@ class Whisper {
     });
   }
 
+  shouldInit() {
+    return (
+      this.communicationConfig.enabled
+    );
+  }
+
+  _getNodeState(err, version, cb) {
+    if (err) return cb({ name: "Ethereum node not found", status: 'off' });
+
+    let nodeName = "go-ethereum";
+    let versionNumber = version.split("-")[0];
+    let name = nodeName + " " + versionNumber + " (Ethereum)";
+    return cb({ name, status: 'on' });
+  }
+
+  // TODO: need to get correct port taking into account the proxy
```

Comment:
This TODO is not needed, as the ports in the config are fine, they will connect to the proxy
```suggestion

```